plugins/vendor/zoom/lib/model/data/kZoomRecording.php

```diff
@@ -65,6 +65,34 @@ public function parseRecordingFiles($recordingFilesData)
 
 			$this->recordingFiles[$kZoomRecordingFile->recordingStart][$kZoomRecordingFile->recordingFileType][] = $kZoomRecordingFile;
 		}
+		$this->sortZoomRecordingFilesByRecordingTypes();
+	}
+
+	public function sortZoomRecordingFilesByRecordingTypes()
```

Comment:
add a comment that explains the structure of the recording files and why it needs to be reordered

Reply:
done